tools/db_bench_tool.cc

```diff
@@ -604,7 +603,7 @@ DEFINE_bool(use_stderr_info_logger, false,
 
 DEFINE_bool(use_blob_db, false, "Whether to use BlobDB. ");
 
-enum rocksdb::CompressionType StringToCompressionType(const char* ctype) {
+static enum rocksdb::CompressionType StringToCompressionType(const char* ctype) {
```

Comment:
how can functions with internal linkage conflict with functions defined elsewhere? I also don't see where the style guide says what you're suggesting, this section says both are allowed: https://google.github.io/styleguide/cppguide.html#Unnamed_Namespaces_and_Static_Variables